src/test/java/org/springframework/boot/autoconfigure/data/jdbc/support/TestUtilities.java

```diff
@@ -41,4 +46,16 @@ public static AnnotationConfigApplicationContext prepareApplicationContext(Class
 
 		return context;
 	}
+
+	public static <T> T getField(Object source, String name) {
+
+		try {
+
+			Field field = CascadingDataAccessStrategy.class.getDeclaredField(name);
+			field.setAccessible(true);
```

Comment:
Seems like a handy utility, however, it's immediately confined to a single class type. Consider making the class a third argument?

Reply:
Oh ... missed that.